docs/microsoft_tests.md

```diff
@@ -46,11 +40,13 @@ Ideally, all tests should be run to maximize the coverage. But the time and reso
 
 Test tiers can be T0, T1, T2, T3, T4. It maps to priorities of test cases. For example, T0 means all P0 test cases are selected in a test run. T2 means all P0, P1, P2 test cases are selected in a test run.
 
-- **T0** completes in 5 minutes with single VM in average. 100% automation, and no need for manual analysis of results.
-- **T1** completes in 2 hours with 2 VMs in each environment, and two environments are parallel. 100% automation, and no need for manual analysis of results.
-- **T2** completes in 8 hours with two environments are parallel. 100% automation.
-- **T3** completes in 16 hours with two environments are parallel. 100% automation.
-- **T4** has no cost limit, and the coverage is maximized. 100% automation.
+| name | test priorities    | time restriction | resource restriction                    | automation requirement                                       |
```

Comment:
I believe it will be good if we also give some details on what all tests we cover as part of these priorities - p0, p1 etc. Thoughts?

Reply:
Yes, it's in next section of this document. The test cases of v3 are not listed, and the task is in backlog.

Reply:
sounds good.